src/Middleware/RequestBodyBufferMiddleware.php

```diff
@@ -21,7 +22,7 @@
     public function __construct($sizeLimit = null)
     {
         if ($sizeLimit === null) {
-            $sizeLimit = $this->iniMaxPostSize();
+            $sizeLimit = Util::iniSizeToBytes(ini_get('post_max_size'));
         }
 
         $this->sizeLimit = $sizeLimit;
```

Comment:
Note that explicitly giving a size here does not allow size suffixes. Also needs tests?

Reply:
Yeah seems that slipped through, fixed it here :+1: